requests_toolbelt/adapters/appengine.py

```diff
@@ -5,31 +5,40 @@
 """
 import requests
 from requests import adapters
+from requests import sessions
 
 from .. import exceptions as exc
 from .._compat import gaecontrib
+from .._compat import timeout
 
+"""
+There are two ways to use this library.
```

Comment:
This needs to document that it was added in v0.6.0 of the toolbelt and that this is a *submodule*.

```rst
.. versionadded:: 0.6.0
```